runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IndustriesSection/IndustriesSlide/IndustriesSlide.module.scss

```diff
@@ -22,6 +22,7 @@ $SHOW-MORE-RADIUS: 0px 0px 20px 20px;
 
     &__open {
         height: fit-content;
+        box-shadow: $box-shadow-primary;
 
         div:first-child {
             height: $SLIDE-OPEN-IMG;
```

Comment:
unclear var name  $SLIDE-OPEN-IMG -> should contain keyword WIDTH/MARGIN/PADDDING/HEIGHT/etc.

first use as HEIGHT, few lines lower as MARGIN-TOP